dataactvalidator/config/sqlrules/c26_award_financial.sql

```diff
@@ -0,0 +1,229 @@
+    COALESCE(ABS(ussgl488200_upward_adjustm_cpe_sum_c) - ABS(ussgl488200_upward_adjustm_cpe_sum_b), 0) AS "difference_ussgl488200_upward_adjustm_cpe_sum",
+    COALESCE(ABS(gross_outlays_undelivered_fyb_sum_c) - ABS(gross_outlays_undelivered_fyb_sum_b), 0) AS "difference_gross_outlays_undelivered_fyb_sum",
+    COALESCE(ABS(gross_outlays_undelivered_cpe_sum_c) - ABS(gross_outlays_undelivered_cpe_sum_b), 0) AS "difference_gross_outlays_undelivered_cpe_sum",
+    COALESCE(ABS(ussgl490200_delivered_orde_cpe_sum_c) - ABS(ussgl490200_delivered_orde_cpe_sum_b), 0) AS "difference_ussgl490200_delivered_orde_cpe_sum",
+    COALESCE(ABS(ussgl490800_authority_outl_fyb_sum_c) - ABS(ussgl490800_authority_outl_fyb_sum_b), 0) AS "difference_ussgl490800_authority_outl_fyb_sum",
+    COALESCE(ABS(ussgl490800_authority_outl_cpe_sum_c) - ABS(ussgl490800_authority_outl_cpe_sum_b), 0) AS "difference_ussgl490800_authority_outl_cpe_sum",
+    COALESCE(ABS(ussgl498200_upward_adjustm_cpe_sum_c) - ABS(ussgl498200_upward_adjustm_cpe_sum_b), 0) AS "difference_ussgl498200_upward_adjustm_cpe_sum",
+    COALESCE(ABS(gross_outlays_delivered_or_fyb_sum_c) - ABS(gross_outlays_delivered_or_fyb_sum_b), 0) AS "difference_gross_outlays_delivered_or_fyb_sum",
+    COALESCE(ABS(gross_outlays_delivered_or_cpe_sum_c) - ABS(gross_outlays_delivered_or_cpe_sum_b), 0) AS "difference_gross_outlays_delivered_or_cpe_sum",
+    COALESCE(ABS(gross_outlay_amount_by_awa_fyb_sum_c) - ABS(gross_outlay_amount_by_pro_fyb_sum_b), 0) AS "difference_gross_outlay_amount_by_pro_fyb_sum",
+    COALESCE(ABS(gross_outlay_amount_by_awa_cpe_sum_c) - ABS(gross_outlay_amount_by_pro_cpe_sum_b), 0) AS "difference_gross_outlay_amount_by_pro_cpe_sum",
+    COALESCE(ABS(obligations_incurred_byawa_cpe_sum_c) - ABS(obligations_incurred_by_pr_cpe_sum_b), 0) AS "difference_obligations_incurred_by_pr_cpe_sum",
+    COALESCE(ABS(ussgl487100_downward_adjus_cpe_sum_c) - ABS(ussgl487100_downward_adjus_cpe_sum_b), 0) AS "difference_ussgl487100_downward_adjus_cpe_sum",
+    COALESCE(ABS(ussgl497100_downward_adjus_cpe_sum_c) - ABS(ussgl497100_downward_adjus_cpe_sum_b), 0) AS "difference_ussgl497100_downward_adjus_cpe_sum",
+    COALESCE(ABS(ussgl487200_downward_adjus_cpe_sum_c) - ABS(ussgl487200_downward_adjus_cpe_sum_b), 0) AS "difference_ussgl487200_downward_adjus_cpe_sum",
+    COALESCE(ABS(ussgl497200_downward_adjus_cpe_sum_c) - ABS(ussgl497200_downward_adjus_cpe_sum_b), 0) AS "difference_ussgl497200_downward_adjus_cpe_sum",
+    COALESCE(ABS(deobligations_recov_by_awa_cpe_sum_c) - ABS(deobligations_recov_by_pro_cpe_sum_b), 0) AS "difference_deobligations_recov_by_pro_cpe_sum",
+    award_financial_records.display_tas AS "uniqueid_TAS",
+    award_financial_records.disaster_emergency_fund_code AS "uniqueid_DisasterEmergencyFundCode"
+-- This first subquery is selecting the sum of 32 elements in File C based on TAS, DEFC, and Submission ID
+FROM (
```

Comment:
These should probably be written as `WITH`s rather than as subqueries. Partially for readability and partially for performance reasons because the CTE is executed exactly once but subqueries can be executed multiple times depending on the situation.